easybuild/easyconfigs/b/brotli/brotli-1.0.9-GCC-8.3.0.eb

```diff
@@ -0,0 +1,35 @@
+# easybuild easyconfig
+#
+# John Dey <jfdey@fredhutch.org>
+#
+# Fred Hutchinson Cancer Research Center - Seattle Washington - US
+#
+easyblock = 'CMakeMake'
+name = 'brotli'
```

Comment:
I think we should go for `Brotli` as the name here

Reply:
Brotli-1.0.9-GCC-8.3.0.eb